packages/mobile/src/send/saga.ts

```diff
@@ -267,19 +272,18 @@ function* sendPayment(
       JSON.stringify(feeInfo)
     )
 
-    // TODO: Add temporary tx to feed.
-    // yield put(
-    //   addStandbyTransaction({
-    //     context,
-    //     type: TokenTransactionType.Sent,
-    //     comment,
-    //     status: TransactionStatus.Pending,
-    //     value: amount.toString(),
-    //     tokenAddress,
-    //     timestamp: Math.floor(Date.now() / 1000),
-    //     address: recipientAddress,
-    //   })
-    // )
+    yield put(
+      addStandbyTransaction({
+        context,
+        type: TokenTransactionTypeV2.Sent,
+        comment,
+        status: TransactionStatus.Pending,
+        value: amount.toString(),
```

Comment:
I _think_ we need to add a `-` here, since this is an outgoing transaction, the value change is negative. I think this is an issue with the legacy stuff as well. Don't worry about the legacy (not sure if it's even a problem), but for the new feed we need it negative I think.

Reply:
Make sense, I will change it